_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+We welcome young packages that are just entering the scientific Python ecosystem to apply for review if they are relevant to the science community and fit into at least one scope category below. We also welcome mature packages with a growing or established community!
+
+The following categories are in-scope for the pyOpenSci review process:
+* **Data retrieval:** refers to packages for accessing and downloading data from online sources, including wrappers for accessing APIs.
+* **Data extraction:** these packages aid in retrieving data from unstructured sources such as text, images, and PDFs. They might also parse scientific data types and outputs from scientific equipment.
+* **Data processing/munging:** this category focuses on tools for handling data in specific formats that scientists may be interested in working with.
+* **Data deposition:** includes tools for depositing data into scientific research repositories.
+* **Data validation and testing:** refers to tools that enable automated validation and checking of data quality and completeness that in turn support scientific workflows.
+* **Data visualization:** these are packages that enhance a scientist’s experience in visualizing and analyzing data. If your package falls into the category of data visualization, please fill out a [pre-submission inquiry](https://github.com/pyOpenSci/software-submission/issues/new?assignees=&labels=0%2Fpresubmission&template=presubmission-inquiry.md&title=) first.
+* **Workflow automation:** these tools automate and link together workflows, and as such support reproducible workflows. These tools may include build systems and tools to manage continuous integration, as well as support version control.
+* **Citation management and bibliometrics:** these are packages that facilitate managing references.
+* **Scientific software wrappers:** these tools provide a Python interface for existing scientific packages written in other languages.
+* **Database interoperability:** these packages deal with bindings and wrappers for database APIs.
+
+Similar to the previous section, you can indicate which categories are applicable to your package in one of two ways:
+* adding an "X" (without quotation marks) inside the brackets in the Markdown version of the issue, so that it reads like this: [X]
+* submitting the issue without making any changes to this section, and then clicking on the checkboxes when they're rendered in the submitted issue.
+
+You can read more about each of these categories in the [Package Scope](https://www.pyopensci.org/software-peer-review/index.html) section of the [pyOpenSci Peer Review Guide](https://www.pyopensci.org/software-peer-review/about/package-scope.html).
+
+### Domain specific
```

Comment:
So i think where i'm getting hung up here is that the headers are the sections in the template ... i think breaking this down into steps would be really helpful

for example:
`### Step x: Select any scope categories that relate to your package`

There might be other alternatices but i'm looking for a way to add stronger structure and more descriptive / useful headings to this page.